internal/api/front/perses/src/app/project/dashboard/dashboard-details/dashboard-details.component.ts

```diff
@@ -56,4 +106,28 @@ export class DashboardDetailsComponent implements OnInit {
       },
     );
   }
+
+  private convertDashboardFeeds(dashboardFeeds: DashboardFeedModel[]): void {
+    for (const section of dashboardFeeds) {
+      for (const panel of section.panels) {
+        const ngxPanelData: NgxLineChartModel[] = []
+        for (const query of panel.results) {
+          let i = 0;
+          for (const serie of query.result) {
+            const ngxSerieData: NgxLineChartModel = {
+              name: serie.metric.__name__ + " - " + i,
+              series: []
+            };
+            for (const [timestamp, value] of serie.values) {
+              const datapoint: NgxPoint = {name: String(timestamp), value: Number(value)};
+              ngxSerieData.series.push(datapoint)
+            }
+            ngxPanelData[i] = ngxSerieData
```

Comment:
Some semicolon are missing in the file, I guess it is already reported by the linter

Reply:
yes sorry, I pinged you too soon actually, fixed since